src/components/Counter.js

```diff
@@ -0,0 +1,32 @@
+import { Component } from "react";
+import React from "react";
+
+export default class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.initialValue,
+    };
+
+    this.increment = this.increment.bind(this);
+    this.decrement = this.decrement.bind(this);
```

Comment:
Good job. We can also use [arrow function expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) to avoid having to bind the methods in the constructor.